### ISSUE TITLE
Conf.rb gem

### ISSUE THREAD

**Issue by gsbhasin123** [markets/awesome-ruby#962]
Hey, I really wanted my gem https://rubygems.org/gems/confrb to be added. It's a very simple plaintext configuration library that supports JSON and YAML and is heavily based on classes.

**Comment by markets** [comment]
Hi @gsbhasin123 and thanks a lot for sharing your project. 

A couple of comments:
- We already have a lot of config gems (https://github.com/markets/awesome-ruby#configuration), what's the main difference compared to "competitors" or alternatives?
- Seems pretty fresh for this list, only ~1k downloads (we usually take in consideration gems above 10k downloads)

**Comment by gsbhasin123** [comment]
Thanks, @markets! 
* Unlike the others (at least from what I can tell), my gem is much simpler and based on a class approach. It allows a user to easily read/write to files in plaintext, JSON, or YAML.
* Ah I see. Can I leave this issue opem until it hits 10-11k downloads/

**Comment by markets** [comment]
> Can I leave this issue opem until it hits 10-11k downloads?

yes of course, this is what we usually do, even with PRs where the community is encouraged to add +1, -1, comments, etc.

**Comment by gsbhasin123** [comment]
Great!

**Comment by gsbhasin123** [comment]
Yay, 9,463 downloads ;)
Hopefully I hit 10-11k soon

**Comment by IpProxyNeon** [comment]
I think this is a really good gem, and I've enjoyed using it quite a bit for its simplicity, tho I am still new so :P
This gem is really good tho, especially for a beginner at Ruby like me

**Comment by gsbhasin123** [comment]
10,540 now! It's surprising how fast I can get downloads.
